### PR TITLE
Add some redirects for broken links or missing documentation

### PR DESCRIPTION
**Pull request by stefanroelofs**
Add some redirects to fix broken links in documentation.
See commit messages for more details.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_redirects [hunk 1]
```diff
@@ -511,8 +514,9 @@ layout: null
 /integrations/htu21d /more-info/removed-integration 301
 /integrations/huawei_router /more-info/removed-integration 301
 /integrations/hydroquebec /more-info/removed-integration 301
-/integrations/ialarm /more-info/removed-integration 301
```
Comment (frenck) [inline source/_redirects:514]: We need to keep ialarm as well, it exists at some point.

Comment (stefanroelofs) [reply source/_redirects:514]: Well, no: The iAlarm integration is still functional and included in Home Assistant.  Documentation is here: https://www.home-assistant.io/integrations/ialarm/

(So the redirect does not seem to be functioning? I don't know why)

Comment (frenck) [reply source/_redirects:514]: Oh right, that was added back later! 👍 

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @stefanroelofs 👍 

../Frenck

**Pull request merged**